app/src/main/java/org/love2d/android/LoveDocumentsProvider.java

```diff
@@ -0,0 +1,713 @@
+/*
+ * Copyright (C) 2013 The Android Open Source Project
+ *
```

Comment:
I think this should have an extra line saying there are modifications, maybe something like (if you want to put your contributions under the name of love developers):
```
Modified for use with LOVE by the LOVE Development Team
Copyright (C) 2022 LOVE Development Team
```

Or maybe something like this, if you don't:
```
Modified for use with LOVE by Vixeliz
Copyright (C) 2022 Vixeliz
```

(I'm sure the language used could be changed a bit if you want, that's just what came to mind.)

Reply:
Yeah I'll do that, I'll just use the love developers